src/plugins/intel_cpu/src/nodes/multiclass_nms.cpp

```diff
@@ -106,15 +261,23 @@ void MultiClassNms::initSupportedPrimitiveDescriptors() {
     const std::vector<Precision> supportedFloatPrecision = {Precision::FP32, Precision::BF16};
     const std::vector<Precision> supportedIntOutputPrecision = {Precision::I32, Precision::I64};
 
-    checkPrecision(getOriginalInputPrecisionAtPort(NMS_BOXES), supportedFloatPrecision, "boxes", m_inType);
-    checkPrecision(getOriginalInputPrecisionAtPort(NMS_SCORES), supportedFloatPrecision, "scores", m_inType);
+    const char* inTypeStr = "input";
+    const char* outTypeStr = "output";
+
+    auto check_precision = [&](const Precision prec, const std::vector<Precision> precList, const std::string name, const char* type) {
+        if (std::find(precList.begin(), precList.end(), prec) == precList.end())
+            IE_THROW() << err_str_prefix_ << "has unsupported '" << name << "' " << type << " precision: " << prec;
+    };
+
+    check_precision(getOriginalInputPrecisionAtPort(NMS_BOXES), supportedFloatPrecision, "boxes", inTypeStr);
+    check_precision(getOriginalInputPrecisionAtPort(NMS_SCORES), supportedFloatPrecision, "scores", inTypeStr);
 
-    checkPrecision(getOriginalOutputPrecisionAtPort(NMS_SELECTEDINDICES), supportedIntOutputPrecision, "selected_indices", m_outType);
-    checkPrecision(getOriginalOutputPrecisionAtPort(NMS_SELECTEDOUTPUTS), supportedFloatPrecision, "selected_outputs", m_outType);
-    checkPrecision(getOriginalOutputPrecisionAtPort(NMS_SELECTEDNUM), supportedIntOutputPrecision, "selected_num", m_outType);
+    check_precision(getOriginalOutputPrecisionAtPort(NMS_SELECTEDINDICES), supportedIntOutputPrecision, "selected_indices", outTypeStr);
+    check_precision(getOriginalOutputPrecisionAtPort(NMS_SELECTEDOUTPUTS), supportedFloatPrecision, "selected_outputs", outTypeStr);
```

Comment:
Please, use the original style of naming for `check_precision` like `checkPrecision`

Reply:
done